### ISSUE TITLE
testkit frontmatter comments does not parse on Windows

### ISSUE THREAD

**Issue by olafurpg** [scalacenter/scalafix#364]
After the Scala World presentation, @szeiger had problems running the tests from the g8 generate template on his Windows laptop.

@szeiger, can you maybe add the error message?

**Comment by MasseGuillaume** [comment]
```
lmeyer @LeonardMeyer 11:54
[IJ]> unit/testOnly scalafix.tests.rule.*
[info] SemanticTests:
[info] - scala/test/ImplicitClassPrivateParams.scala *** FAILED *** (100 milliseconds)
[info]   java.lang.IllegalStateException: parseerror: String: 3: Reserved character '*' is not allowed outside quotes (if you intended '*' (Reserved character '*' is not allowed outside quotes) to be part of a key or string value, try enclosing the key or value in double quotes)
[info]   at metaconfig.Configured.get(Configured.scala:11)
[info]   at scalafix.testkit.DiffTest$$anonfun$$nestedInanonfun$fromSemanticdbIndex$2$1.applyOrElse(DiffTest.scala:41)
[info]   at scalafix.testkit.DiffTest$$anonfun$$nestedInanonfun$fromSemanticdbIndex$2$1.applyOrElse(DiffTest.scala:33)
[info]   at scala.collection.TraversableOnce.collectFirst(TraversableOnce.scala:145)
[info]   at scala.collection.TraversableOnce.collectFirst$(TraversableOnce.scala:132)
[info]   at scala.meta.tokens.Tokens.collectFirst(Tokens.scala:28)
[info]   at scalafix.testkit.DiffTest$.$anonfun$fromSemanticdbIndex$2(DiffTest.scala:33)
[info]   at scalafix.testkit.SemanticRuleSuite.$anonfun$runOn$1(SemanticRuleSuite.scala:73)
[info]   at org.scalatest.OutcomeOf.outcomeOf(OutcomeOf.scala:85)
[info]   at org.scalatest.OutcomeOf.outcomeOf$(OutcomeOf.scala:83)
[info]   ...
```

https://gitter.im/scalacenter/scalafix?at=5a574268ae53c15903d4f2e2

**Comment by MasseGuillaume** [comment]
https://github.com/scalameta/scalameta/issues/1208

**Comment by MasseGuillaume** [comment]
Workaround:

```
// DiffTest.scala
private def stripPrefix(str: String) = {
    val lb = 13.toChar
    PrefixRegex.replaceFirstIn(str, "").stripSuffix("*/" + lb)
  }
```

**Comment by MasseGuillaume** [comment]
@LeonardMeyer
https://github.com/MasseGuillaume/scalafix/commit/9e78a80679765ef5d91c7c79977e12cce7dadd31

Then you want to run:
`~unit/testOnly scalafix.tests.rule.SemanticTests`

Remove `ONLY` when you are ready for the review.

**Comment by olafurpg** [comment]
I believe this is fixed in #737, we no longer use `Token.Comment.unapply`
